app/views/admin/lodgings/index.html.haml

```diff
@@ -12,7 +12,11 @@
         .thumbnail
           - unless lodging.picture?
             %p.text-center
-              %i.fa.fa-home.fa-5x
+              - if lodging.website_link.present?
+                = link_to(lodging.website_link, :style => 'color:inherit') do
```

Comment:
do not add style to the link, but a class, to avoid defining style here. 😉

Reply:
Sure @Ana06. I am adding a gif for your reference as it will be more clear with the gif. Here for the lodging i am using youtube url.
![lodging-link](https://user-images.githubusercontent.com/24351012/34621986-46a8fa82-f271-11e7-8a84-a4a388a17a10.gif)

And I want to ask whether the link color should be blue(as default) or no color for link?

Reply:
Actually it was not needed, I though you were changing how it looks like, not the behaviour. Sorry, I misinterpreted. 🙈 Thanks for the gift anyway 😉

Reply:
Please add the style to a class and not inline 😉  I think we want that it has no color, it would be strange in blue, right?